### PR TITLE
Edit: Add Codebyte to Javascript .getTime()

### LINKED ISSUE #4139: [Edit] Dates: Adding codebyte for the entry `.getTime()`
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to the getTime entry. The term entry lives in Codecademy/docs/content/javascript/concepts/dates/terms/getTime/getTime.md.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a codebyte which shows how we can use `.getTime()`.
2) You can be creative with the codebyte snippet but make sure you don't deviate a lot from the entry.

### Codebyte Example

image

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

Add a Codebyte to demonstrate the .getTime() method in Javascript.

Closes #4139 

### Type of Change

- Editing an existing entry (adding a Codebyte)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/getTime/getTime.md [hunk 1]
```diff
@@ -35,3 +35,14 @@ This will yield:
 ```shell
 1672531200000
 ```
+
+## Codebyte Example
+
+Below is a runnable codebyte which defines `today` as today's date and then converts it into Unix time by using the `.getTime()` method. A `console.log` then prints the value:
```
Comment (Sriparno08) [inline content/javascript/concepts/dates/terms/getTime/getTime.md:41]:
```suggestion
Below is a runnable codebyte which defines `today` as today's date and then converts it into milliseconds by using the `.getTime()` method. A `console.log()` then prints the value:
```

**Review by Sriparno08 — COMMENTED** [review]
Hey @Jamesllllllllll, just a single change from my side:

**Comment by Jamesllllllllll** [comment]
Thanks @Sriparno08 - made the changes.

**Review by Sriparno08 — APPROVED** [review]
Looks good for a second review 👍

**Review by SSwiniarski — APPROVED** [review]
@Jamesllllllllll Looks good.

**Pull request merged**